sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ClientSideRequestStatistics.java

```diff
@@ -129,7 +127,12 @@ public void recordGatewayResponse(
             }
             this.gatewayStatistics = new GatewayStatistics();
             if (rxDocumentServiceRequest != null) {
+                if (rxDocumentServiceRequest.getResourceType().equals(ResourceType.Document)) {
```

Comment:
Can you include a before and after in the PR description to show the changes to the JSON output?